src/SmoFacade/Database.cs

```diff
@@ -77,12 +77,7 @@ public List<BackupMetadata> RecentBackups()
                   "s.database_backup_lsn, s.checkpoint_lsn, s.[type] AS backup_type, s.server_name, s.recovery_model " +
                   "FROM msdb.dbo.backupset s " +
                   "INNER JOIN msdb.dbo.backupmediafamily m ON s.media_set_id = m.media_set_id " +
-                  "WHERE s.last_lsn >= (" +
-                  "SELECT MAX(last_lsn) FROM msdb.dbo.backupset " +
-                  "WHERE [type] = 'D' " +
-                  "AND database_name = @dbName " +
-                  "AND is_copy_only = 0" +
-                  ") " +
+                  "WHERE s.backup_start_date > DATEADD(day, -30, GETDATE())" +
```

Comment:
This might work for us, but since we are outsourcing this, it might not work for everyone.
It might be better if we parameterise this to allow customisation?

Reply:
Yeah I've thought about that.  In our case, 7 would suffice. I chose 30 as a safe(r) timerange
I'd rather not add a parameter, the alternative IMO is what I linked in PR # 97